src/components/StudyInfo/FileCounts.js

```diff
@@ -1,47 +1,83 @@
 import React from 'react';
-import {List, Icon, Popup, Label} from 'semantic-ui-react';
+import {Link} from 'react-router-dom';
+import {List, Icon, Popup, Label, Button} from 'semantic-ui-react';
 import {versionState} from '../../common/fileUtils';
 
-const FileCounts = ({files}) => {
-  const states = files.map(
-    ({node: {versions}}) => versions.edges[0].node.state,
-  );
-  const stateCounts = states.reduce((count, state) => {
-    count[state] = (count[state] || 0) + 1;
-    return count;
-  }, {});
+const FileCounts = ({files, title, history}) => {
+  if (files && files.length > 0) {
+    const states = files.map(
+      ({node: {versions}}) => versions.edges[0].node.state,
+    );
+    const stateCounts = states.reduce((count, state) => {
+      count[state] = (count[state] || 0) + 1;
+      return count;
+    }, {});
 
-  return (
-    <List horizontal>
-      <List.Item>
-        <Icon name="file" />
-        {files.length} files
-      </List.Item>
-      {['PEN', 'CHN', 'APP', 'PRC'].map(
-        state =>
-          state in stateCounts && (
-            <Popup
-              inverted
-              position="top center"
-              size="small"
-              content={versionState[state].title}
-              key={state}
-              trigger={
-                <List.Item>
-                  <Label
-                    circular
-                    empty
-                    size="mini"
-                    color={versionState[state].labelColor}
-                  />{' '}
-                  {stateCounts[state]}
-                </List.Item>
-              }
-            />
-          ),
-      )}
-    </List>
-  );
+    return (
+      <List horizontal>
+        <List.Item
+          as={Link}
+          to={`/study/${title}/documents`}
+          onClick={e => e.stopPropagation()}
+        >
+          <Icon name="file" />
+          {files.length} files
+        </List.Item>
+        {['PEN', 'CHN', 'APP', 'PRC'].map(
+          state =>
+            state in stateCounts && (
+              <Popup
+                inverted
+                position="top center"
+                size="small"
+                content={versionState[state].title}
+                key={state}
+                trigger={
+                  <List.Item>
+                    <Label
+                      circular
+                      empty
+                      size="mini"
+                      color={versionState[state].labelColor}
+                    />{' '}
+                    {stateCounts[state]}
+                  </List.Item>
+                }
+              />
+            ),
+        )}
+      </List>
+    );
+  } else {
+    return (
+      <List horizontal>
```

Comment:
same as CavaticaCounts, could maybe move the list up and conditioanlly render the the list items to make the file shorter

Reply:
We can move this up but that would make the study card file and study table file a lot longer. I think we can keep it here in case in the future we want to reuse it somewhere else.